noise: 10.0
string_var: abc
vector_var: [1,2,3]
a_list: [4,5,6]